modules/instruments/test/TestPlan.md

```diff
@@ -0,0 +1,18 @@
+# Instrument Test Plan
+
+1. Verify that in order to access the module, the user must either have the `Across all sites access
+candidate profiles` permission, or have the 'Edit Candidate Parameters' permission and be at the same
+site as the visit, or at the same site as one of the candidate's other visits.
+2. Select and instrument in the list of the session instruments. Note that the visit's stage should
+be at 'Visit'.
+3. Ensure that the candidate info at the top (DoB, EDC, ...) are correct.
+In the instrument's medical history section, enter a date in the future and click on the 'save data'
```

Comment:
thats good but remove mention to medical history specifically. any instruments has the "date of administration" field. stay generic